apps/preview/next/pages/showcases/MegaMenu/BaseMegaMenu.tsx

```diff
@@ -0,0 +1,253 @@
+                  aria-expanded={isOpen}
+                  slotSuffix={
+                    <>
+                      <SfIconChevronRight className="hidden rotate-90 md:inline-flex" />
+                      <SfIconMenu className="inline-flex md:hidden" />
+                    </>
+                  }
+                  variant="tertiary"
+                  onClick={toggle}
+                >
+                  <span className="hidden md:inline-flex">Browse products</span>
+                </SfButton>
+                <CSSTransition
+                  ref={drawerRef}
+                  in={isOpen}
+                  timeout={500}
+                  unmountOnExit
+                  classNames={{
+                    enter: '-translate-x-full md:opacity-0 md:translate-x-0',
+                    enterActive: 'translate-x-0 md:opacity-100 transition duration-500 ease-in-out',
+                    exitDone: 'translate-x-0 md:opacity-100',
+                    exitActive: '-translate-x-full md:opacity-0 transition duration-500 ease-in-out',
+                  }}
+                >
+                  <SfDrawer
+                    open={isOpen}
+                    placement="top"
+                    className="grid grid-cols-1 gap-6 md:grid-cols-4 bg-white max-w-xs shadow-lg p-0 !fixed md:!absolute md:!top-[5rem] md:max-w-full md:p-8"
+                  >
+                    <div className="flex items-center justify-between py-2 px-4 bg-primary-700 md:hidden">
+                      <div className="flex items-center typography-text-lg font-medium text-white">Browse products</div>
+                      <SfButton
+                        square
+                        variant="tertiary"
+                        aria-label="Close navigation menu"
+                        onClick={close}
+                        className="text-white"
+                      >
+                        <SfIconClose />
+                      </SfButton>
+                    </div>
+                    {categoriesContent.map(({ heading, items }) => (
+                      <div key={heading}>
+                        <h2
+                          role="presentation"
+                          className="typography-text-xs font-bold text-neutral-900 tracking-widest whitespace-nowrap px-4 py-2"
+                        >
+                          {heading}
+                        </h2>
+                        <hr className="mb-3.5" />
+                        <ul>
+                          {items.map((item) => (
+                            <SfListItem size="sm" role="none" key={item.title}>
+                              <a
+                                className="focus-visible:outline focus-visible:outline-offset focus-visible:rounded-sm"
+                                href={item.link}
+                              >
+                                {item.title}
+                              </a>
+                            </SfListItem>
```

Comment:
To make the whole area of the ListItem clickable you can wrap it with an anchor.
```suggestion
                            <a
                              key={item.title}
                              className="focus-visible:outline focus-visible:rounded-sm"
                              href={item.link}
                            >
                              <SfListItem size="sm" role="none">
                                {item.title}
                              </SfListItem>
                            </a>
```
I suggest you remove `outline-offset` class as it overlaps other items in this case.

Reply:
@dkacper actually we cant do that, <li> can be the only descendant of <ul> for a11y/semantic reasons.